modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -263,36 +267,57 @@ const RichText = ({
 		>
 			<ClayInput.Group>
 				<ClayInput.GroupItem>
-					<ClassicEditor
-						ariaLabel={label}
-						ariaRequired={otherProps.required}
-						className="w-100"
-						contents={
-							currentValue
-								? currentValue[currentEditingLocale?.localeId]
-								: ''
-						}
-						editorConfig={editorConfig}
-						name={name}
-						onBlur={onBlur}
-						onChange={(content) => handleContentChange(content)}
-						onFocus={onFocus}
-						onSetData={(event) => {
-							const editor = event.editor;
+					{Liferay.FeatureFlags['LPD-11235'] ? (
+						<CKEditor5ClassicEditor
+							className="w-100"
+							config={{
+								...advancedClassicEditorConfig,
```

Comment:
This is more than fine now.

I'd like to open a small discussion about how to configure this editor. At some point in the future, this configuration could be provided via `editorConfig` just like `<ClassicEditor>` does. But for that to happen, we'd need to apply the same FF at the point where such config contributors are invoked, so that the branching applies not only to the editor flavor but also to configs.

This is how I see it however I want to check whether you guys have a different opinion on this.

Any options I missed?

Reply:
Yeah, you are right, and I will need to implement a mechanism like that in the very next PR. Integration with DL requires a set of URLs, that come from backend configuration.